example_project/example_project/orchestra_settings.py

```diff
@@ -57,6 +57,11 @@ def setup_orchestra(settings_module_name):
         ('text/x-scss', 'django_libsass.SassCompiler'),
     )
 
+    # Pass global variables (e.g., Google Analytics key) to templates with
+    # this context processor.
+    settings.TEMPLATES[0]['OPTIONS']['context_processors'].append(
+        'orchestra.context_processors.global_settings')
```

Comment:
Two comments:
* Is this global_settings trick something that others have done / a common design pattern in Django? It feels a bit odd to me...
* https://docs.djangoproject.com/en/1.9/ref/settings/#templates suggests that the default setting is an empty list--we probably need to check for that and add a default entry ourselves.